modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/comparator/StructureLinkStructureNameComparator.java

```diff
@@ -24,8 +24,14 @@
 public class StructureLinkStructureNameComparator
 	extends OrderByComparator<DDMStructureLink> {
 
-	public StructureLinkStructureNameComparator(boolean ascending) {
-		this(ascending, LocaleUtil.getDefault());
+	public static StructureLinkStructureNameComparator getInstance(
+		boolean ascending) {
+
+		if (ascending) {
+			return _ASCENDING;
+		}
+
+		return _DESCENDING;
 	}
 
 	public StructureLinkStructureNameComparator(
```

Comment:
@MrJohn1911 It seems this constructor has no external usage, either. Please double check.